keadm/cmd/keadm/app/cmd/debug/check.go

```diff
@@ -285,7 +285,7 @@ func CheckNetWork(IP string, timeout int, cloudhubServer string, edgecoreServer
 	if config != "" {
 		edgeConfig, err := util.ParseEdgecoreConfig(config)
 		if err != nil {
-			err = fmt.Errorf("parse Edgecore config failed")
+			return fmt.Errorf("parse Edgecore config failed")
 		} else {
```

Comment:
You can delete the else here.

if err != nil {
			return fmt.Errorf("parse Edgecore config failed")
		} 

if cloudhubServer == "" {
				cloudhubServer = edgeConfig.Modules.EdgeHub.WebSocket.Server
......

Reply:
Sorry，I thought you ask me delete whole else part. Get it now.

Reply:
The code still need gofmted.

Reply:
the extra spaces have been cleaered